cogs/mod.py

```diff
@@ -170,109 +170,38 @@ def check_if_target_is_staff(self, target):
         userlog(target.id, ctx.author, reason, "bans", target.name)
 
         safe_name = await commands.clean_content().convert(ctx, str(target))
-
-        dm_message = f"You were banned from {ctx.guild.name}."
-        if reason:
-            dm_message += f" The given reason is: \"{reason}\"."
-        dm_message += "\n\nThis ban does not expire."
-
-        try:
-            await target.send(dm_message)
-        except discord.errors.Forbidden:
-            # Prevents ban issues in cases where user blocked bot
-            # or has DMs disabled
-            pass
+        
+        ban_type = "Ban"
+        if ctx.invoked_with in ["softban","silentban"]:
+            ban_type = "Silent Ban"
+            dm_message = f"You were banned from {ctx.guild.name}."
+            if reason:
+                dm_message += f" The given reason is: \"{reason}\"."
+            dm_message += "\n\nThis ban does not expire."
+
+            try:
+                await target.send(dm_message)
+            except discord.errors.Forbidden:
+                # Prevents ban issues in cases where user blocked bot
+                # or has DMs disabled
+                pass
 
         await target.ban(reason=f"{ctx.author}, reason: {reason}",
                          delete_message_days=0)
-        chan_message = f"⛔ **Ban**: {ctx.author.mention} banned "\
+        chan_message = f"⛔ **{ban_type}**: {ctx.author.mention} banned "\
                        f"{target.mention} | {safe_name}\n"\
                        f"🏷 __User ID__: {target.id}\n"
         if reason:
             chan_message += f"✏️ __Reason__: \"{reason}\""
         else:
             chan_message += "Please add an explanation below. In the future"\
-                            ", it is recommended to use `.ban <user> [reason]`"\
+                           f", it is recommended to use `.{ctx.invoked_with} <user> [reason]`"\
                             " as the reason is automatically sent to the user."
 
         log_channel = self.bot.get_channel(config.modlog_channel)
         await log_channel.send(chan_message)
         await ctx.send(f"{safe_name} is now b&. 👍")
```

Comment:
Same as +177 -> +187 comment here, though if you'd like you can include this line in the aforementioned if block too to result in a bit cleaner code.

Reply:
My apologies, but can you expand on this?